src/main/scala/eventstore/operations/Operation.scala

```diff
@@ -42,6 +42,7 @@ private[eventstore] object Operation {
       case Unsubscribe          => Some(simple(UnsubscribeInspection))
       case ScavengeDatabase     => Some(simple(ScavengeDatabaseInspection))
       case Authenticate         => Some(simple(AuthenticateInspection))
+      case x: IdentifyClient    => Some(simple(ClientIdentifiedInspection))
```

Comment:
@t3hnar I am not sure what I am supposed to do anything else than an simple `ClientIdentifiedInspection` here

Reply:
I think it should be `None`, similar to `Pong`